semperfiwebdesign/all-in-one-seo-pack
# Trigger sitemap notice only when XML Sitemap module is active

Issue #2756

## Proposed changes

Fixes an issue where our sitemap notice is triggered even when the XML Sitemap module isn't enabled.

## Types of changes

- Bugfix (non-breaking change which fixes an issue)

## Checklist


## Testing instructions

Enable the XML Sitemap module, then enable Sitemap Indexes and set the amount of pages per index to something larger than 1,000. The notice should appear (if this is on a fresh test site). Now dismiss the notice and disable the XML Sitemap module again. Then go to the database, delete aioseop_notices and make sure the notice isn't triggered again. 

I believe this only happens when people have previously used our XML Sitemap module and the their default amount of pages was set to something larger than 1,000.
